File: ver_py/T12.py
# https://leetcode.cn/problems/a-number-after-a-double-reversal/submissions/
class Solution:
    def isSameAfterReversals(self, num: int) -> bool:
        return num==0 or num % 10 > 0
if __name__ == "__main__":
    num = int(input())
    print(Solution().isSameAfterReversals(num))

File: ver_cpp/T12.cpp
// https://leetcode.cn/problems/a-number-after-a-double-reversal/submissions/
#include<bits/stdc++.h>
using namespace std;

class Solution {
public:
    bool isSameAfterReversals(int num) {
        return num==0 || num%10>0;
    }
};

int main(){
    int num;
    cin>>num;
    Solution s;
    cout<<s.isSameAfterReversals(num)<<endl;
}